src/pages/billing/Billing.ts

```diff
@@ -0,0 +1,28 @@
+import { List } from 'src/pages/billing/List/List'
+
+
```

Comment:
I decided to leave a top-level file here for exporting the nav path and storing a couple of utilities that are used by multiple files.

Reply:
I did starting out with `billing-utils`, but had a feeling that we are trying to avoid dumping grounds like that. And my main motivation here was to have the navPath exporting in a top-level file (matching the name of the navPath) vs. down in `pages/billing/List/List`.

@nawatts and/or @msilva-broad what are your thoughts about these issues? It seems like we have a variety of different approaches currently in the codebase, and I'm not sure what our goal is here.

Reply:
I'd vote for billing-utils (at least for billingRoles and billingProjectNameValidator). I think that more closely matches existing naming conventions (such as they are). Base on naming conventions, in a file name Billing, I'd expect to find a component named Billing.

Where I've really seen the "utils becomes a dumping ground" is in the generic "utils" and where multiple components start getting added to a utils module, such as in data-utils. I think between keeping it specific to "billing utils" and our more recent tendency towards smaller components and splitting components into separate modules, a "utils" module is fine.

As for navPaths, if it were up to me, all the navPaths would live in one routes file so that it would be easier to find which component mapped to a URL. But that's a break from existing conventions, so I won't argue for that here.

Reply:
OK, I will move the utilities to billing-utils, and I will move the navPath section to the actual file that declares the component (`pages/billing/List/List`).